### ISSUE TITLE
Lottery Simulator example doesn't work with version 0.2.2 - tabs don't show

### ISSUE THREAD

**Issue by BorisNaguet** [dart-lang/angular2_components#46]
Hello,

I've followed all the example but with the latest Dart/Angular2/angular2_components.
Everything works except for the last step when the divs are replaced by "material-tab".
To be sure that I made no mistake I replaced the code by the "step-4" from github and it still doesn't work.
I managed to run the "component example" page with the same versions, and the tabs are ok, so there must just be something missing in the demo.

Steps to reproduce:

```shell
git clone https://github.com/dart-lang/one-hour-codelab.githttps://github.com/dart-lang/one-hour-codelab.git
cd one-hour-codelab\angular2_components\4-final\
pub get
pub deps
pub serve
```

Here everything is working :)

```shell
vi pubspec.yaml

pub get
Resolving dependencies...
> angular2 2.2.0 (was 2.1.1)
> angular2_components 0.2.2 (was 0.2.0)
Changed 2 dependencies!

pub deps
pub serve
```

Here it doesn't work...

**Comment by BorisNaguet** [comment]
for information:
```shell
dart --version
Dart VM version: 1.21.0 (Wed Dec 07 06:52:21 2016) on "windows_x64"
```

The 1st "deps" returned:
```
components_codelab 0.0.1
|-- angular2 2.1.1
|   |-- analyzer 0.29.1
|   |   |-- args...
|   |   |-- charcode 1.1.1
|   |   |-- crypto...
|   |   |-- glob...
|   |   |-- html...
|   |   |-- isolate 0.2.3
|   |   |-- meta...
|   |   |-- package_config 1.0.0
|   |   |   |-- charcode...
|   |   |   '-- path...
|   |   |-- path...
|   |   |-- plugin 0.2.0
|   |   |-- watcher...
|   |   '-- yaml...
|   |-- barback...
|   |-- build 0.5.0
|   |   |-- analyzer...
|   |   |-- barback...
|   |   |-- code_transformers 0.5.0+1
|   |   |   |-- analyzer...
|   |   |   |-- barback...
|   |   |   |-- cli_util 0.0.1+2
|   |   |   |   |-- path...
|   |   |   |   '-- which 0.1.3
|   |   |   |       |-- path...
|   |   |   |       '-- when 0.2.0
|   |   |   |-- path...
|   |   |   |-- source_maps 0.10.1+2
|   |   |   |   |-- path...
|   |   |   |   '-- source_span...
|   |   |   '-- source_span...
|   |   |-- crypto...
|   |   |-- glob...
|   |   |-- logging...
|   |   |-- path...
|   |   |-- shelf 0.6.7+1
|   |   |   |-- async...
|   |   |   |-- collection...
|   |   |   |-- http_parser...
|   |   |   |-- path...
|   |   |   |-- stack_trace...
|   |   |   '-- stream_channel 1.5.0
|   |   |       |-- async...
|   |   |       '-- stack_trace...
|   |   |-- shelf_static 0.2.4
|   |   |   |-- convert...
|   |   |   |-- http_parser 3.0.3
|   |   |   |   |-- collection...
|   |   |   |   |-- source_span...
|   |   |   |   '-- string_scanner...
|   |   |   |-- mime 0.9.3
|   |   |   |-- path...
|   |   |   '-- shelf...
|   |   |-- stack_trace...
|   |   |-- watcher 0.9.7+3
|   |   |   |-- async...
|   |   |   |-- collection...
|   |   |   '-- path...
|   |   '-- yaml 2.1.12
|   |       |-- charcode...
|   |       |-- collection...
|   |       |-- source_span...
|   |       '-- string_scanner...
|   |-- dart_style 0.2.13
|   |   |-- analyzer...
|   |   |-- args 0.13.7
|   |   |-- path...
|   |   '-- source_span...
|   |-- glob 1.1.3
|   |   |-- async...
|   |   |-- collection...
|   |   |-- path...
|   |   '-- string_scanner 1.0.1
|   |       |-- charcode...
|   |       '-- source_span...
|   |-- html...
|   |-- intl...

|   |-- logging 0.11.3+1
|   |-- observable...
|   |-- path 1.4.0
|   |-- protobuf 0.5.3
|   |   '-- fixnum 0.10.5
|   |-- source_span 1.3.0
|   |   '-- path...
|   '-- stack_trace 1.7.0
|       '-- path...
|-- angular2_components 0.2.0
|   |-- angular2...
|   |-- intl...
|   |-- js 0.6.1
|   |-- observable 0.14.0+1
|   |   '-- meta 1.0.4
|   |-- perf_api 0.1.0

|   |   '-- matcher 0.12.0+2
|   '-- uuid 0.5.3
|       |-- convert 2.0.1
|       |   |-- charcode...
|       |   '-- typed_data 1.1.3
|       '-- crypto 2.0.1
|           |-- collection...
|           '-- convert...
|-- browser 0.10.0+2
|-- dart_to_js_script_rewriter 1.0.2
|   |-- barback 0.15.2+9
|   |   |-- async 1.11.3
|   |   |   '-- collection...
|   |   |-- collection 1.9.1
|   |   |-- path...
|   |   |-- pool 1.2.4
|   |   |   |-- async...
|   |   |   '-- stack_trace...
|   |   |-- source_span...
|   |   '-- stack_trace...
|   '-- html 0.13.0
|       |-- csslib 0.13.2+2
|       |   |-- args...
|       |   |-- logging...
|       |   |-- path...
|       |   '-- source_span...
|       |-- source_span...
|       '-- utf 0.9.0+3
'-- intl 0.14.0
```

and then:
```
components_codelab 0.0.1
|-- angular2 2.1.1
|   |-- analyzer 0.29.1
|   |   |-- args...
|   |   |-- charcode 1.1.1
|   |   |-- crypto...
|   |   |-- glob...
|   |   |-- html...
|   |   |-- isolate 0.2.3
|   |   |-- meta...
|   |   |-- package_config 1.0.0
|   |   |   |-- charcode...
|   |   |   '-- path...
|   |   |-- path...
|   |   |-- plugin 0.2.0
|   |   |-- watcher...
|   |   '-- yaml...
|   |-- barback...
|   |-- build 0.5.0
|   |   |-- analyzer...
|   |   |-- barback...
|   |   |-- code_transformers 0.5.0+1
|   |   |   |-- analyzer...
|   |   |   |-- barback...
|   |   |   |-- cli_util 0.0.1+2
|   |   |   |   |-- path...
|   |   |   |   '-- which 0.1.3
|   |   |   |       |-- path...
|   |   |   |       '-- when 0.2.0
|   |   |   |-- path...
|   |   |   |-- source_maps 0.10.1+2
|   |   |   |   |-- path...
|   |   |   |   '-- source_span...
|   |   |   '-- source_span...
|   |   |-- crypto...
|   |   |-- glob...
|   |   |-- logging...
|   |   |-- path...
|   |   |-- shelf 0.6.7+1
|   |   |   |-- async...
|   |   |   |-- collection...
|   |   |   |-- http_parser...
|   |   |   |-- path...
|   |   |   |-- stack_trace...
|   |   |   '-- stream_channel 1.5.0
|   |   |       |-- async...
|   |   |       '-- stack_trace...
|   |   |-- shelf_static 0.2.4
|   |   |   |-- convert...
|   |   |   |-- http_parser 3.0.3
|   |   |   |   |-- collection...
|   |   |   |   |-- source_span...
|   |   |   |   '-- string_scanner...
|   |   |   |-- mime 0.9.3
|   |   |   |-- path...
|   |   |   '-- shelf...
|   |   |-- stack_trace...
|   |   |-- watcher 0.9.7+3
|   |   |   |-- async...
|   |   |   |-- collection...
|   |   |   '-- path...
|   |   '-- yaml 2.1.12
|   |       |-- charcode...
|   |       |-- collection...
|   |       |-- source_span...
|   |       '-- string_scanner...
|   |-- dart_style 0.2.13
|   |   |-- analyzer...
|   |   |-- args 0.13.7
|   |   |-- path...
|   |   '-- source_span...
|   |-- glob 1.1.3
|   |   |-- async...
|   |   |-- collection...
|   |   |-- path...
|   |   '-- string_scanner 1.0.1
|   |       |-- charcode...
|   |       '-- source_span...
|   |-- html...
|   |-- intl...

|   |-- logging 0.11.3+1
|   |-- observable...
|   |-- path 1.4.0
|   |-- protobuf 0.5.3
|   |   '-- fixnum 0.10.5
|   |-- source_span 1.3.0
|   |   '-- path...
|   '-- stack_trace 1.7.0
|       '-- path...
|-- angular2_components 0.2.0
|   |-- angular2...
|   |-- intl...
|   |-- js 0.6.1
|   |-- observable 0.14.0+1
|   |   '-- meta 1.0.4
|   |-- perf_api 0.1.0

|   |   '-- matcher 0.12.0+2
|   '-- uuid 0.5.3
|       |-- convert 2.0.1
|       |   |-- charcode...
|       |   '-- typed_data 1.1.3
|       '-- crypto 2.0.1
|           |-- collection...
|           '-- convert...
|-- browser 0.10.0+2
|-- dart_to_js_script_rewriter 1.0.2
|   |-- barback 0.15.2+9
|   |   |-- async 1.11.3
|   |   |   '-- collection...
|   |   |-- collection 1.9.1
|   |   |-- path...
|   |   |-- pool 1.2.4
|   |   |   |-- async...
|   |   |   '-- stack_trace...
|   |   |-- source_span...
|   |   '-- stack_trace...
|   '-- html 0.13.0
|       |-- csslib 0.13.2+2
|       |   |-- args...
|       |   |-- logging...
|       |   |-- path...
|       |   '-- source_span...
|       |-- source_span...
|       '-- utf 0.9.0+3
'-- intl 0.14.0
```

**Comment by kwalrath** [comment]
I'll update the code lab to request more narrow versions of angular2 & angular2_components.

cc @filiph

**Comment by nshahan** [comment]
I am able to reproduce this issue with the currently released versions:
angular2: 2.2.0
angular2_components: 0.2.2

I am investigating a fix. It appears to be an interaction between material tabs and material expansion panels.

**Comment by nshahan** [comment]
I have a fix that I hope to be in the next components release.

**Comment by nshahan** [comment]
The fix is not in the latest releases but it is still coming.

**Comment by nshahan** [comment]
Fix for this issue landed in #57. 

@kwalrath already locked down the versions of the examples in the one-hour-codelab repo.
